pyproject.toml

```diff
@@ -30,6 +30,7 @@ dependencies = [
     "onnxruntime==1.16.3",
     "numpy==1.23.0",
     "protobuf==3.20.2",
+    "peft==0.12.0",
```

Comment:
@quic-jhugo @carlstreeter-quic, Could you pls review, this is consolidated PR for all the package version changes in one place.

Reply:
This doesn't seem to have the changes for PR80.  Are those still planned?

Reply:
It looks like this uses torch==2.0.0 rather than torch==2.0.0+cpu.  This results in pulling the cuda packages which have unclear licensing and will not be approved.  This will not work.